scripts/grafana-server/custom.ini

```diff
@@ -4,7 +4,7 @@ content_security_policy = true
 content_security_policy_template = """require-trusted-types-for 'script'; script-src 'self' 'unsafe-eval' 'unsafe-inline' 'strict-dynamic' $NONCE;object-src 'none';font-src 'self';style-src 'self' 'unsafe-inline' blob:;img-src * data:;base-uri 'self';connect-src 'self' grafana.com ws://$ROOT_PATH wss://$ROOT_PATH;manifest-src 'self';media-src 'none';form-action 'self';"""
 
 [feature_toggles]
-enable = publicDashboards
+enable = publicDashboards, pinNavItems
```

Comment:
Since [this PR](https://github.com/grafana/grafana/pull/94194) will turn the feature toggle on by default, we should remove it but waiting for you to review it; otherwise, the test will fail in your repro.